scripts/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+cargo check --all-targets --all-features --locked
+cargo build --locked --all-targets --all-features
```

Comment:
Something I ran into before, [suprisingly](https://github.com/rust-lang/cargo/issues/6669) `--all-targets` doesn't run doc tests, whereas `cargo test` would. I don't think we have any examples in the docs, but may be worth adding `cargo test --docs ...` too so we don't forget.

Also, suuupper nit. the arguments could be in same order

Reply:
Good point - yes, I feel like I've been annoyed about that before.

Have un-nitted the arguments, and removed some duplicate/redundant flags (like, we don't need --all-features because we have no features at the moment).